modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,44 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.3
+
+* The {SMProductName} Operator is now available on ARM based clusters as a Technology Preview feature.
```

Comment:
```suggestion
* The {SMProductName} Operator is now available on ARM-based clusters as a Technology Preview feature.
```
Nitpick: It should be `ARM-based clusters` with a hyphen throughout this section, unless this wording (with no hyphen) is following some already-established naming convention for the feature.

Reply:
@bburt-rh made the change and sent you a DM